cve_bin_tool/checkers/bluez.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+"""
+CVE checker for Bluez
+References:
+https://www.cvedetails.com/vulnerability-list/vendor_id-8316/product_id-35116/Bluez-Bluez.html
+
+"""
+from ..util import regex_find
+
+# def get_version(lines, filename):
+def get_version(lines, filename):
+    """Bluetoothctl will work for Version 5.0+"""
```

Comment:
You'll need to include the
Vpkg: bluez, bluez 

string in the bottom of this docstring so that the checker can do the magic of associating it with the database entries.

Reply:
I added the Vpkg string at the bottom of the doc string. I might have made an unneeded comment, so you might need to squash the commits. Thanks for the guidance!